First draft (#639)
OctopusDeploy/blog: blog/2022-01/gitlab-external-feeds/index.md

---
title: Using the GitLab feeds with Octopus Deploy
description: Learn how to connect GitLab feed types to Octopus Deploy.
author: shawn.sesna@octopus.com
visibility: private 
published: 2022-12-06-1400
metaImage: blogimage-usinggitlabfeedswithoctopus-2022.png
bannerImage: blogimage-usinggitlabfeedswithoctopus-2022.png

> bannerImageAlt: Packages on conveyer belt starting in an orange machine, traveling through a blue machine, coming out the other side with version numbers.

Reviewer: ```suggestion
bannerImageAlt: Packages on conveyer belts start in an orange machine, travel through a blue machine, and come out deployed with version numbers.
```
